File: portal/src/main/resources/static/service/finance/taxService.js
function taxService(PATIENT_MODULE_CONFIG,MODULE_CONFIG, $http) {
	this.addtax = function(tax) {
	return 	$http.post(PATIENT_MODULE_CONFIG.ADD_TAX(),tax);
	};
	this.getAlltaxs=function(){
		return $http.get(PATIENT_MODULE_CONFIG.GETALLTAX());
	};
	this.getAlltaxList=function(email,page, size){
		return $http.get(PATIENT_MODULE_CONFIG.GETALLTAXSLIST(email,page, size));
	};
	this.updatetax=function(tax){
		return $http.put(PATIENT_MODULE_CONFIG.UPDATETAX(),tax);
	};
	this.deletetax=function(id){
		return $http.delete(PATIENT_MODULE_CONFIG.DELETETAX(id));
	};
	
}
angular.module('HealthApplication').service('taxService', taxService);

File: patient/src/main/java/com/openspace/PatientManagement/service/TaxService.java
package com.openspace.PatientManagement.service;

import java.util.List;

import org.springframework.data.domain.Page;

import com.openspace.Model.DoctorManagement.Tax;

public interface TaxService {

	void addTax(Tax tax);

	List<Tax> getAllTaxs();

	void updateTax(Tax tax);

	void deleteTax(Long id);

	List<Tax> getAllTaxsByAdmin(String username);

	Page<Tax> getAllTaxsByAdminPage(String username, int page, int size);
}
